jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/CheckpointManager.scala

```diff
@@ -1,5 +1,5 @@
 /*
- Copyright (c) 2014 by Contributors
+ Copyright (c) 2014 - 2019 by Contributors
```

Comment:
can we remove header changes, as I said, there are more to fix around this, 

the current header is not recognized in scala style checker, we need to look into more about that

Reply:
will do

Reply:
done